cumulusci/tasks/salesforce/activate_flow.py

```diff
@@ -0,0 +1,53 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+from cumulusci.core.exceptions import TaskOptionsError
+
+
+class ActivateFlow(BaseSalesforceApiTask):
+    """
+    Activate the Flows with the supplied Developer Names
+    """
+
+    task_options = {
+        "developer_names": {
+            "description": "List of DeveloperNames to query in SOQL",
+            "required": True,
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super(ActivateFlow, self)._init_options(kwargs)
+        self.options["developer_names"] = process_list_arg(
+            self.options.get("developer_names")
+        )
+        self.api_version = "43.0"
+        if not self.options["developer_names"]:
```

Comment:
This will never be `False` because it's initialized in lines 20-21 (and the option is required). Tip: you can recognize unneeded code like this because you will never be able to cover it in a unit test.

Reply:
I think it is needed other wise I get `File "/Users/jkofsky/Documents/python_3.7/CumulusCI/cumulusci/core/tasks.py", line 140, in __call__
    self.result = self._run_task()
  File "/Users/jkofsky/Documents/python_3.7/CumulusCI/cumulusci/tasks/salesforce/activate_flow.py", line 36, in _run_task
    ",".join([f"'{n}'" for n in self.options["developer_names"]])
TypeError: 'NoneType' object is not iterable` but if this is acceptable I'm ok with that, this way handled in a more user friendly way in my opinion.

Reply:
Sorry, I need to fully engage my brain before reviewing code. It shouldn't be `None`, but it may be _empty_ (`[]`) after `process_list_arg()`, which you'd still want to warn the user about.